executor/write_test.go

```diff
@@ -1087,3 +1087,22 @@ func assertEqualStrings(c *C, got []string, expect []string) {
 		c.Assert(got[i], Equals, expect[i])
 	}
 }
+
+// Test issue https://github.com/pingcap/tidb/issues/4067
```

Comment:
Is this comment not needed? Because we can directly infer it from the function name.